ansible/files/queries.yml

```diff
@@ -0,0 +1,27 @@
+pg_database:
+  query: "SELECT SUM(pg_database_size(pg_database.datname)) / (1024 * 1024) as size_mb FROM pg_database"
```

Comment:
aha - this is great saves us fetching from their DB each time

Reply:
Thanks for the pointer--will bring them in here

Reply:
I just realised that this is the generic "postgres" image (which anyone can use), so some of the `auth/storage` schema metrics will be irrelevant..